composer.json

```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "silverstripe/framework": "^4.0"
+        "silverstripe/framework": ">= 4.2"
```

Comment:
Not sure how I feel about this. Wouldn't this promise compatibility with SS 5/6/7 etc.? A limitation of Composer (If I understand correctly) is that if I tag a release with this in it, I can't ever go back and update that tag if it turns out the code at that point is only compatible with SS4. I could tag a new release with a tighter version constraint, but previous release(s) would still exist and be seen as a viable options for SS5+ by Composer. I think ^4.2 would be safer?

Reply:
Yeah, bonehead move here. That should be `^4.2`